ArmaforcesMissionBot/Handlers/ArchiveHandler.cs

```diff
@@ -59,6 +59,8 @@ private async void CheckMissionsToArchiive(object source, ElapsedEventArgs e)
                             .AddField("Zamknięcie zapisów:", mission.CloseTime.ToString())
                             .WithAuthor(_client.GetUser(mission.Owner).Username)
                             .AddField("Modlista:", mission.Modlist);
+							
+						archiveEmbed.Author.Url = BotConstants.DISCORD_USER_URL_PREFIX + (mission.Owner).ToString();	
```

Comment:
1) Formatting is broken

2) You can add url as named param in `WithAuthor()` method. Please add this parameter in separate line. The same applies to SignupHelper.